docs/spec/v1beta1/provider.md

```diff
@@ -155,3 +155,43 @@ metadata:
 data:
   token: <username>:<app-password>
 ```
+
+### Generic webhook
+
+The `generic` webhook triggers an HTTP POST request to the provided endpoint.
+
+The `Gotk-Component` header identifies which component this event is coming
+from, e.g. `source-controller`, `image-reflector-controller`.
```

Comment:
image-reflector-controller is not yet included in Flux, can you please change it to kustomize-controller please?